lib/inspec/utils/telemetry/collector.rb

```diff
@@ -8,7 +9,12 @@ class Collector
 
```

Comment:
I suggest exposing @config via attr_reader, so we avoid the ivar accesses (which are vulnerable to typos)